test/utils/arch.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"runtime"
+)
+
+// IsRunningOnArm64 returns if tests is currently running on an arm64 machine
+func IsRunningOnArm64() bool {
```

Comment:
nit: We only call this in one place, do we actually need a new public util here?

Reply:
Fair enough, I will remove this public util for now.